src/averageAges.js

```diff
@@ -35,6 +52,24 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const womens = people.filter(person => person.sex === 'f');
+
+  const womenWithChildren = people.filter(person =>
+    people.some(child => child.mother === person.name));
```

Comment:
Краще не розривати колбек, падає читабельність
```suggestion
  const womenWithChildren = people.filter(
    person => people.some(child => child.mother === person.name));
```